src/createServer.js

```diff
@@ -1,10 +1,45 @@
 'use strict';
 
+const http = require('http');
+const fs = require('fs');
+const path = require('path');
+
 function createServer() {
-  /* Write your code here */
-  // Return instance of http.Server class
+  return http.createServer((req, res) => {
+    const url = new URL(req.url, `http://${req.headers.host}`);
+    const absolutePaths = path.resolve(url.pathname);
+
+    res.setHeader('Content-Type', 'text/plain');
+
+    if (!absolutePaths.includes('file')) {
+      res.statusCode = 400;
+      res.end();
+    } else if (!url.pathname.startsWith('/file/')) {
+      res.statusCode = 200;
+      res.write('Path should starts with /folder/');
```

Comment:
The message 'Path should starts with /folder/' is misleading since the code checks for '/file/'. Ensure that your messages accurately reflect the conditions being checked.